cmd/tailscale/cli/file.go

```diff
@@ -449,9 +443,62 @@ func runFileGet(ctx context.Context, args []string) error {
 		}
 		deleted++
 	}
-	if getArgs.verbose {
+	if deleted == 0 && len(wfs) > 0 {
+		// persistently stuck files are basically an error
+		logf("moved %d/%d files", deleted, len(wfs))
+	} else if getArgs.verbose {
 		printf("moved %d/%d files\n", deleted, len(wfs))
 	}
+	return errs
+}
+
+func runFileGet(ctx context.Context, args []string) error {
+	if len(args) != 1 {
+		return errors.New("usage: file get <target-directory>")
+	}
+	log.SetFlags(0)
```

Comment:
no need to do this every loop? move out earlier?

Reply:
not sure i follow.

Reply:
ah, whoops. I misread!